docs/source/guides/inference.rst

```diff
@@ -59,3 +59,44 @@ As an example, OpenVINO inference can be triggered by the following command:
 ``python tools/inference/openvino.py --config padim.yaml --weights results/openvino/model.xml --input image.png --meta_data results/openvino/meta_data.json``
 
 Similar to PyTorch inference, the visualization results will be displayed on the screen, and optionally saved to the file system location specified by the ``save_data`` parameter.
+
+
+
+Gradio Inference
+==============
+
+The gradio inference is supported for both PyTorch and OpenVINO models.
+
++-----------+----------+------------------------------------------------------------------+
+| Parameter | Required |                           Description                            |
++===========+==========+==================================================================+
+| config    | True     | Path to the model config file.                                   |
++-----------+----------+------------------------------------------------------------------+
+| weights   | True     | Path to the OpenVINO IR model file (either ``.xml`` or ``.bin``) |
++-----------+----------+------------------------------------------------------------------+
+| meta_data | False    | Path to the JSON file containing the model's meta data.          |
+|           |          | This is needed only for OpenVINO model.                          |
++-----------+----------+------------------------------------------------------------------+
+| threshold | False    | Threshold value used for identifying anomalies. Range 1-100.     |
++-----------+----------+------------------------------------------------------------------+
+| share     | False    | Share Gradio `share_url`                                         |
++-----------+----------+------------------------------------------------------------------+
+
+To use gradio with OpenVINO model, first make sure that your model has been exported to the OpenVINO IR format and ensure that the `meta_data` argument points to the ``meta_data.json`` file that was generated when exporting the OpenVINO IR model. The file is stored in the same folder as the ``.xml`` and ``.bin`` files of the model.
+
+As an example, PyTorch model can be used by the following command:
+
+.. code-block:: bash
+
+    python tools/inference/gradio_inference.py \
+        --config ./anomalib/models/padim/config.yaml \
+        --weights ./results/padim/mvtec/bottle/weights/model.ckpt
```

Comment:
Maybe another PR, but I think we should use the lightning inference to handle the meta-data automatically. Otherwise, we manually need to tune the threshold on the UI